R/annotation-map.r

```diff
@@ -59,13 +59,13 @@ NULL
 annotation_map <- function(map, ...) {
   # Get map input into correct form
   if (!is.data.frame(map)) {
-    abort("`map` must be a data.frame")
+    cli::cli_abort("{.arg map} must be a {.cls data.frame}")
   }
   if (!is.null(map$lat)) map$y <- map$lat
   if (!is.null(map$long)) map$x <- map$long
   if (!is.null(map$region)) map$id <- map$region
   if (!all(c("x", "y", "id") %in% names(map))) {
-    abort("`map`must have the columns `x`, `y`, and `id`")
+    cli::cli_abort("{.arg map} must have the columns {.col x}, {.col y}, and {.col id}")
```

Comment:
I wonder if `{.col {c("x", "y", "id")}}` is worth it? Or maybe pull out into separate variable?

Reply:
no strong opinions... spelling it out makes it a bit clearer what the message will end up looking like...